exec/java-exec/src/main/resources/options.describe.properties

```diff
@@ -0,0 +1,106 @@
+planner.memory.percent_per_query=Sets the memory as a percentage of the total direct memory.
+planner.nestedloopjoin_factor=A heuristic value for influencing the nested loop join.
+planner.partitioner_sender_max_threads=Upper limit of threads for outbound queuing.
+planner.partitioner_sender_set_threads=Overwrites the number of threads used to send out batches of records. Set to -1 to disable. Typically not changed.
+planner.partitioner_sender_threads_factor=A heuristic param to use to influence final number of threads. The higher the value the fewer the number of threads.
+planner.producer_consumer_queue_size=How much data to prefetch from disk in record batches out-of-band of query execution. The larger the queue size, the greater the amount of memory that the queue and overall query execution consumes.
+planner.slice_target=The number of records manipulated within a fragment before Drill parallelizes operations.
+planner.width.max_per_node=Maximum number of threads that can run in parallel for a query on a node. A slice is an individual thread. This number indicates the maximum number of slices per query for the query’s major fragment on a node.
+planner.width.max_per_query=Same as max per node but applies to the query as executed by the entire cluster. For example, this value might be the number of active Drillbits, or a higher number to return results faster.
+security.admin.user_groups=Unsupported as of 1.4. A comma-separated list of administrator groups for Web Console security.
+security.admin.users=Unsupported as of 1.4. A comma-separated list of user names who you want to give administrator privileges.
+store.format=Output format for data written to tables with the CREATE TABLE AS (CTAS) command. Allowed values are parquet, json, psv, csv, or tsv.
+store.hive.optimize_scan_with_native_readers=Optimize reads of Parquet-backed external tables from Hive by using Drill native readers instead of the Hive Serde interface. (Drill 1.2 and later)
+store.json.all_text_mode=Drill reads all data from the JSON files as VARCHAR. Prevents schema change errors.
+store.json.extended_types=Turns on special JSON structures that Drill serializes for storing more type information than the four basic JSON types.
+store.json.read_numbers_as_double=Reads numbers with or without a decimal point as DOUBLE. Prevents schema change errors.
+store.mongo.all_text_mode=Similar to store.json.all_text_mode for MongoDB.
+store.mongo.read_numbers_as_double=Similar to store.json.read_numbers_as_double.
+store.parquet.block-size=Sets the size of a Parquet row group to the number of bytes less than or equal to the block size of MFS, HDFS, or the file system.
+store.parquet.compression=Compression type for storing Parquet output. Allowed values: snappy, gzip, none
+store.parquet.enable_dictionary_encoding=For internal use. Do not change.
+store.parquet.dictionary.page-size=
```

Comment:
description?

Reply:
Picked this up from the website, so I guess this slipped through. :)